map/routing_manager.hpp

```diff
@@ -259,6 +259,12 @@ class RoutingManager final
   {
     m_routingSession.SetTurnNotificationsUnits(units);
   }
+
+  void SetIsLandscape(bool isLandscape)
+  {
+    m_routingSession.SetIsLandscape(isLandscape);
```

Comment:
Is there an easier way to check that the map is in landscape mode using the Framework? E.g. compare the dimensions of the map screen?

CC @vng

Reply:
There are some spots in the code that compared dimensions, and some that used the system. Either way, I'm pretty sure that data will have to be passed through to the routingsession, so the logic should just need to be changed at the top. Note too, the dimensions will still need to be checked every time the orientation changes, so it needs to be at the top of the call stack either way

Reply:
Please check if `void Framework::OnViewportChanged` is called on landscape/portrait rotation, and if yes, then landscape flag can be passed from there.

Reply:
Ooh nice, yeah putting it there (and removing it from the other spots) updates it correctly. I'll commit the change shortly